### PR TITLE
Fix analytics expander in singlepass

### PR DESCRIPTION
**Pull request by zhouyx**
The expander is broken with singlepass.

### CODE REVIEW

File: src/service/url-expander/expander.js [hunk 1]
```diff
@@ -295,11 +295,11 @@ export class Expander {
   evaluateBinding_(bindingInfo, opt_args) {
     const {encode, name} = bindingInfo;
     let binding;
-    if (hasOwn(bindingInfo, 'prioritized')) {
+    if (bindingInfo.prioritized) {
```
Comment (calebcordry) [inline src/service/url-expander/expander.js]: looks like these need to check explicitly for `undefined` to handle falsy cases.

Comment (zhouyx) [reply src/service/url-expander/expander.js]: Interesting. I thought `bingingInfo.prioritized` is always a function. When will it be a false value?

Comment (calebcordry) [reply src/service/url-expander/expander.js]: macros don't have to be functions. Sometimes they are just static values. Callers can pass in any optional macros as desired e.g. `{FOO: false}`.

Comment (zhouyx) [reply src/service/url-expander/expander.js]: Thanks. But is there a real use case for that?

Comment (calebcordry) [reply src/service/url-expander/expander.js]: I think something like this, where they are passing primitives to the expander:
https://github.com/ampproject/amphtml/blob/e92f42d1a4cb7fc76bd0db0e5282e07f7701d097/extensions/amp-a4a/0.1/real-time-config-manager.js#L138-L141

Comment (zhouyx) [reply src/service/url-expander/expander.js]: I see. Thanks
Any concern if we restrict the macros to resolvers to always be functions?

**Pull request merged**